dcli/generatorCli.py

```diff
@@ -123,3 +125,16 @@ def python(self):
         commonEndingMessage(answers)
         pass
 
+    def kotlin(self):
+        """ Generates a basic Kotlin challenge """
+        answers = askUsual()
+        answers.update(askKotlinQuestions())
+
+        language = Kotlin()
+        self.addTypeIfNecessary(self.parseTargetMethodArgs(answers['targetMethodArgs']), language)
```

Comment:
`addTypeIfNecessary` and `parseTargetMethodArgs` support only Java at this time, don't include this line.

Reply:
Done